hypervisor/include/dm/vacpi.h

```diff
@@ -26,6 +26,10 @@
 #define UOS_VIRT_PCI_MMCFG_BASE		0xE0000000UL
 #define UOS_VIRT_PCI_MMCFG_START_BUS	0x0U
 #define UOS_VIRT_PCI_MMCFG_END_BUS	0xFFU
+#define UOS_VIRT_PCI_MEMBASE32      0x80000000UL    /* 2GB */
+#define UOS_VIRT_PCI_MEMLIMIT32     0xC0000000UL    /* 3GB */
```

Comment:
I think this should change to 0xE0000000UL /**3.5 GB/. Same with the value which is defined in acrn-dm.

Reply:
Fixed, change UOS_VIRT_PCI_MEMLIMIT32 to 3.5GB